### PR TITLE
Create breaking-changes-3.21.adoc

### PR DESCRIPTION
**Pull request by sebdevaux**
Create breaking-changes-3.21.adoc: Docker non root user

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/am-docker-non-root/index.html)

### CODE REVIEW

**Review by passionne — APPROVED** [review]

**Review by remisultan — CHANGES_REQUESTED** [review]
The PR should target am-3.21 branch

File: pages/am/3.x/breaking-changes/breaking-changes-3.21.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+:page-sidebar: am_3_x_sidebar
+:page-permalink: am/current/am_breaking_changes_3.21.html
+:page-folder: am/installation-guide
+:page-layout: am
+
+NOTE: To take advantage of these new features and incorporate these breaking changes, **use the migration guide available link:{{ '/am/current/am_installguide_migration.html' | relative_url }}[here]**.
+
+== Docker images
+
+From this version, to be complient with https://www.tenable.com/audits/items/CIS_Docker_v1.3.1_L1_Docker_Linux.audit:bdcea17ac365110218526796ae3095b1[CIS_Docker_v1.3.1_L1] the docker images are now using the user `graviteeio`
```
Comment (remisultan) [inline pages/am/3.x/breaking-changes/breaking-changes-3.21.adoc:11]:
```suggestion
From this version, to be compliant with https://www.tenable.com/audits/items/CIS_Docker_v1.3.1_L1_Docker_Linux.audit:bdcea17ac365110218526796ae3095b1[CIS_Docker_v1.3.1_L1] the docker images now use the `graviteeio` user
```

File: pages/am/3.x/breaking-changes/breaking-changes-3.21.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+This means that if you use the official images and deploy them to Kubernetes, nothing changes.
+If you build your own Dockerfile from Gravitee images, you must give the right rights to the modifications you add.
```
Comment (remisultan) [inline pages/am/3.x/breaking-changes/breaking-changes-3.21.adoc:13]:
```suggestion
If you build your own Dockerfile from Gravitee images, you must give the correct rights according to your modifications.
```

File: pages/am/3.x/breaking-changes/breaking-changes-3.21.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+If you deploy in `openshift`, you have to add the following configure:
```
Comment (remisultan) [inline pages/am/3.x/breaking-changes/breaking-changes-3.21.adoc:14]:
```suggestion
If you deploy on `openshift`, you have to add the following configuration:
```

File: pages/am/3.x/breaking-changes/breaking-changes-3.21.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+:page-permalink: am/current/am_breaking_changes_3.21.html
+:page-folder: am/installation-guide
+:page-layout: am
+
+NOTE: To take advantage of these new features and incorporate these breaking changes, **use the migration guide available link:{{ '/am/current/am_installguide_migration.html' | relative_url }}[here]**.
+
+== Docker images
+
+From this version, to be complient with https://www.tenable.com/audits/items/CIS_Docker_v1.3.1_L1_Docker_Linux.audit:bdcea17ac365110218526796ae3095b1[CIS_Docker_v1.3.1_L1] the docker images are now using the user `graviteeio`
+This means that if you use the official images and deploy them to Kubernetes, nothing changes.
```
Comment (remisultan) [inline pages/am/3.x/breaking-changes/breaking-changes-3.21.adoc:12]:
```suggestion
This means that if you use the official images and deploy them on your k8s installation, nothing changes.
```

**Review by remisultan — CHANGES_REQUESTED** [review]

**Pull request merged**